README.md

```diff
@@ -43,7 +43,7 @@ If you are on Fedora, you can install the packaged version of the plugin by runn
 <a name="usage"></a>
 ## Usage
 
-The plugin is designed in an registration-manager-agnostic way which means that plugin itself does not depend on any OS nor way of registration. vagrant-registration only calls registration capabilities for given guest, passes the configuration options to them and handles interactive registration.
+The plugin is designed in a registration-manager-agnostic way which means that plugin itself neither depend on any OS nor way of registration. The vagrant-registration only calls registration capabilities for given guest, passes the configuration options to them and handles interactive registration.
 
 That being said, this plugin currently ships only with registration capability files for RHEL's Subscription Manager and `rhn_register`. Feel free to submit others.
 
```

Comment:
To configure the plugin, always include the configuration options mentioned in this file within the following configuration block in your Vagrantfile. 
It is not clear which file we refer to when we say ''this file'', can we be more specific?